packages/server-wallet/src/models/channel.ts

```diff
@@ -162,7 +191,7 @@ export class Channel extends Model implements RequiredColumns {
     }
   }
 
-  signAndAdd(stateVars: StateVariables, privateKey: string): SignedState {
+  signAndAdd(stateVars: StateVariables): SyncState {
```

Comment:
It seems odd to me that a model should need to return a data type related to messaging. Maybe it should still return a `SignedState` and the caller should handle deciding what messages should be queued up?

Reply:
Fair point. This method can be extracted into a pure function that transforms a signed state into a notification. That probably makes more sense